src/main/java/com/mraof/minestuck/tileentity/redstone/AreaEffectTileEntity.java

```diff
@@ -143,15 +202,15 @@ public void load(BlockState state, CompoundNBT compound)
 		
 		effectAmplifier = compound.getInt("effectAmplifier");
 		
-		int minEffectPosX = compound.getInt("minEffectPosX");
-		int minEffectPosY = compound.getInt("minEffectPosY");
-		int minEffectPosZ = compound.getInt("minEffectPosZ");
-		this.minEffectPos = new BlockPos(minEffectPosX, minEffectPosY, minEffectPosZ);
+		int minAreaOffsetX = compound.getInt("minAreaOffsetX");
+		int minAreaOffsetY = compound.getInt("minAreaOffsetY");
+		int minAreaOffsetZ = compound.getInt("minAreaOffsetZ");
+		this.minAreaOffset = new BlockPos(minAreaOffsetX, minAreaOffsetY, minAreaOffsetZ);
 		
-		int maxEffectPosX = compound.getInt("maxEffectPosX");
-		int maxEffectPosY = compound.getInt("maxEffectPosY");
-		int maxEffectPosZ = compound.getInt("maxEffectPosZ");
-		this.maxEffectPos = new BlockPos(maxEffectPosX, maxEffectPosY, maxEffectPosZ);
+		int maxAreaOffsetX = compound.getInt("maxAreaOffsetX");
+		int maxAreaOffsetY = compound.getInt("maxAreaOffsetY");
+		int maxAreaOffsetZ = compound.getInt("maxAreaOffsetZ");
+		this.maxAreaOffset = new BlockPos(maxAreaOffsetX, maxAreaOffsetY, maxAreaOffsetZ);
```

Comment:
Can you make this backwards-compatible? If the nbt tag has an effect pos, it would be loaded and converted to an area offset.

Reply:
Should be fixed now